Tests/SwiftDriverTests/IntegrationTests.swift

```diff
@@ -207,7 +212,7 @@ final class IntegrationTests: IntegrationTestCase {
         "LC_ALL": "en_US.UTF-8"
       ]
 
-      printCommand(args: args, extraEnv: extraEnv)
+      printCommand(args: commandArgs, extraEnv: extraEnv)
 
       let process = TSCBasic.Process(
         arguments: args,
```

Comment:
It seems here we need `commandArgs`, not `args`

```suggestion
        arguments: commandArgs,
```

Reply:
🤦 Changed.